packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -10,6 +10,14 @@ import 'timeline_protocol.dart';
 // Switch this flag to true to dump the frame event trace to console.
 bool _debugEventTrace = false;
 
+// Amber 50 color palette from
+// https://material.io/design/color/the-color-system.html#tools-for-picking-colors.
+const cpuColorPalette = ['#FFECB3', '#FFE082', '#FFD54F', '#FFCA28'];
```

Comment:
Consider using the Color class forked from package:flutter for these. Having a class helps us port to flutter in the future and will help support light + dark themes.

Reply:
Ah yes. Good idea - done.